components/mentor/booking/booking-step-three.tsx

```diff
@@ -0,0 +1,62 @@
+import { Paperclip } from 'lucide-react'
+import { useRef } from 'react'
+
+interface BookingStepThreeProps {
+  selectedFiles: File[]
+  selectFiles: (files: File[]) => void
+  portfolioLink: string
+  handlePortfolioLinkChange?: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+export default function BookingStepThree({
+  selectedFiles,
+  selectFiles,
+  portfolioLink,
+  handlePortfolioLinkChange,
+}: BookingStepThreeProps) {
+  const fileInputRef = useRef<HTMLInputElement>(null)
+  // const [fileNames, setFileNames] = useState<string[]>([])
+
+  const handleFileSelect = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const files = event.target.files
+    if (!files) return
+
+    const newFiles = Array.from(files)
+    selectFiles([...selectedFiles, ...newFiles])
+    // 파일 input 값 초기화해서 같은 파일 선택해도 다시 업로드 가능하도록
+    event.target.value = ''
+  }
```

Comment:
```suggestion
  const handleFileSelect = (event: React.ChangeEvent<HTMLInputElement>) => {
    const files = (...event.target.files ?? [])

    if (!files.length) return

    selectFiles(prev => prev.concat(files))
    // 파일 input 값 초기화해서 같은 파일 선택해도 다시 업로드 가능하도록
    event.target.value = ''
  }
```

요런식으로하면 FileList가 File[] 로 변환되어서 Array 사용안해도 돼요.